app/js/components/names.jsx

```diff
@@ -1,265 +1,241 @@
+              <div className='col-sm-2'>
+                  <input className='btn btn-success form-control'
+                         name='edit'
+                         type='button'
+                         value='Edit Name'
+                         onClick={this.handleEdit}/>
+              </div>
+          </div>}
+
+          <div className='form-group '>
+              <label className='control-label col-sm-2'>Preferred</label>
+              <div className='col-sm-2'>
+                  <input
+                      name='username'
+                      type='checkbox'
+                      checked={this.state.preferred}
+                      disabled={this.state.display === 'view' ?
+                       'disabled' : null}/>
+              </div>
+          </div>
+
+          <div className={'form-group ' + givenErrorClass}>
+              <label className='control-label col-sm-2'>
+              Given Name* </label>
+              <div className='col-sm-5'>
+                  <input  className='form-control'
+                          ref='givenname'
+                          name='givenName'
+                          type='text'
+                          value={this.state.givenName}
+                          onChange={this.handleChange}
+                          readOnly={this.state.display === 'view' ?
+                           'readonly' : null}
+                          required
+                           />
+              </div>
+              {this.state.errorGivenName &&
+              this.state.display === 'edit' &&
+              <div className='input'>{this.state.errorGivenName}</div>}
+          </div>
+
+          <div className='form-group '>
+              <label className='control-label col-sm-2'>
+              Middle Name  </label>
+              <div className='col-sm-5'>
+                  <input  className='form-control'
+                          ref='middlename'
+                          name='middleName'
+                          type='text'
+                          value={this.state.middleName}
+                          onChange={this.handleChange}
+                          readOnly={this.state.display === 'view' ?
+                           'readonly' : null}
+                          />
+              </div>
+          </div>
+
+          <div className={'form-group ' + familyErrorClass}>
+              <label className='control-label col-sm-2'>
+               Family Name* </label>
+              <div className='col-sm-5'>
+                  <input  className='form-control'
+                          ref='familyname'
+                          name='familyName'
+                          type='text'
+                          value={this.state.familyName}
+                          onChange={this.handleChange}
+                          readOnly={this.state.display === 'view' ?
+                           'readonly' : null}
+                          required />
```

Comment:
Please go through this files and fix all indentations

Reply:
Done